tests/scripts/task_python_integration.sh

```diff
@@ -19,29 +19,33 @@
 set -e
 set -u
 
-export PYTHONPATH=python:topi/python:apps/extension/python
```

Comment:
it's better to keep the file untouched. If the we have to set up somthing specifically, we'd better to open a new temporary file

Reply:
Yeah, I think the way we're bypassing CI is hacky right now, not sure what the right approach is.